helper.py

```diff
@@ -1,18 +1,36 @@
 from datetime import datetime
+import os
 import postbin
 
+SQLITE3_ENABLED = True if os.environ['SQLITE3_ENABLED'] == "True" else False
```

Comment:
we could use `True if not 'SQLITE3_ENABLED' in os.environ else os.envior['SQLITE3_ENABLED']` to allow env var not being set